extensions/spark/kyuubi-spark-authz/pom.xml

```diff
@@ -33,6 +33,7 @@
 
     <properties>
         <gethostname4j.version>1.0.0</gethostname4j.version>
+        <jersey.client.version>1.19.4</jersey.client.version>
         <jna.version>5.7.0</jna.version>
     </properties>
```

Comment:
let's move the `ranger.version` from the root pom to this place, and add comments to tips users may need to tune them align w/ `ranger.version`

Reply:
Addressed.